ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/CommandlineParserTest.java

```diff
@@ -237,11 +237,40 @@ void creatingSignerDisplaysFailureToCreateSignerText() {
   }
 
   @Test
-  void missingTlsClientWhitelistIsValid() {
-    missingOptionalParameterIsValidAndMeetsDefault(
-        "tls-known-clients-file",
-        () -> config.getTlsOptions().get().getKnownClientsFile(),
-        Optional.empty());
+  void missingTlsKnownClientFileShowsErrorIfTlsClientAuthenticationIsNotDisabled() {
+    String cmdLine = validBaseCommandOptions();
```

Comment:
Tls Known-Client file is meant to be optional. If client-auth is enabled and client presents a valid CA signed certificate, then there is no need of known-clients file.